test/framework/resources/templates/block-loki-validatingwebhookconfiguration.yaml.tpl

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: block-loki-updates
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    caBundle: {{ .CABundle }}
+    service:
+      name: gardener-seed-admission-controller
```

Comment:
Use something that does not exist. Otherwise this can pollute the logs of the `gardener-seed-admission-controller`.

Reply:
Will it work properly: e.g. to block the update requests for the Loki if I set invalid service name here. 
Also this is copied from the documentation:
```
Note: When using clientConfig.service, the server cert must be valid for <svc_name>.<svc_namespace>.svc.

Reply:
Can't you generate a certificate for a dummy service?